include/sway/commands.h

```diff
@@ -125,7 +126,6 @@ sway_cmd cmd_move;
 sway_cmd cmd_new_float;
 sway_cmd cmd_new_window;
 sway_cmd cmd_no_focus;
-sway_cmd cmd_orientation;
```

Comment:
Why this s/orientation/default_orientation/?

Reply:
The command is named default_orientation, so it was poorly named in the first place.